bin/specextract

```diff
@@ -722,17 +723,16 @@ def set_badpix(evtfile, bpixfile, instrument, verbose):
             raise IOError(f"Failed to set {bpixfile} bad pixel file in ardlib.par for {evtfile}.") from exc
 
     else:
-        bpixpar = f"AXAF_{fileio.get_keys_from_file(evtfile)['DETNAM']}_BADPIX_FILE"
+        detnam = fileio.get_keys_from_file(evtfile)["DETNAM"]
```

Comment:
```suggestion
        detnam = fileio.get_keys_from_file(evtfile)["DETNAM"].replace("-", "_")
```

and then below you can replace `bpixvar.replace(...)` with `bpixvar` (I think there's two of them; I haven't checked closely to know if it breaks anything).